### ISSUE TITLE
Several examples parsed incorrectly

### ISSUE THREAD

**Issue by keskival** [jfromaniello/url-join#31]
https://github.com/jfromaniello/url-join/pull/30

console.log(urljoin('http://example.org', 'a//', 'b//', 'A//', 'B//'))
// -> 'http://example.org/a/b/A//B/'

console.log(urljoin('http://example.org/x', '?a=1', '?b=2', '?c=3', '?d=4'))
// -> http://example.org/x?a=1?b=2?c=3&d=4

console.log(urljoin('http://example.org/', ':foo:', 'bar'))
// -> http://example.org/:foo://bar

**Comment by jfromaniello** [comment]
#30 is merged, let me know if we can close this

**Comment by keskival** [comment]
@jfromaniello : New PR which fixes this and the newly discovered issue: https://github.com/jfromaniello/url-join/pull/35

Once that is merged, then we can close this.

**Comment by keskival** [comment]
And this is now otherwise good, but the package needs to be published to the npm registry, @jfromaniello .
https://www.npmjs.com/package/url-join

**Comment by jfromaniello** [comment]
@keskival sorry about that! just published v3 to npm
